main.go

```diff
@@ -84,3 +90,27 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func checkForKubevirt(kubeClient *kubernetes.Clientset) error {
+	result := kubeClient.ExtensionsV1beta1().RESTClient().Get().RequestURI("/apis/apiextensions.k8s.io/v1/customresourcedefinitions/virtualmachineinstances.kubevirt.io").Do(context.TODO())
+	return result.Error()
+}
+
+// Check for Kubevirt CRD to be available
+func waitForKubevirt() {
+	clientset, err := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
+	if err != nil {
+		setupLog.Error(err, "unable to create a kubernetes client error")
+		os.Exit(1)
+	}
+	for _ = range time.Tick(5 * time.Second) {
+		err := checkForKubevirt(clientset)
+		if err != nil {
+			setupLog.Info("kubevirt doesn't exist in the cluster", "err", err)
```

Comment:
lets print that we are waiting just once in this function start and not every tick please